x-pack/plugins/security/server/security_checkup/check_cluster_data.ts

```diff
@@ -13,16 +13,19 @@ export const createClusterDataCheck = () => {
   return async function doesClusterHaveUserData(esClient: ElasticsearchClient, log: Logger) {
     if (!clusterHasUserData) {
       try {
-        const { indices = {} } = await esClient.indices.stats();
-        const indexIds = indices ? Object.keys(indices) : [];
+        const { indices = {} } = await esClient.indices.stats({
+          index: '*',
+          metric: 'docs',
```

Comment:
__question:__ any reason you prefer to not use [`filter_path`](https://github.com/elastic/kibana/pull/136899#discussion_r932292136) here?